README

```diff
@@ -116,6 +116,23 @@ To install to system run `make install`. By default `make install` will install
     dependencies: *pkg-config*(build-time), *libnl-3* and *libnl-genl-3*
     default: *check*
 
+## runtime dependencies:
+`htop` has a set of fixed runtime dependencies, which is kept as minimal as possible:
+* `ncurses`, libraries for terminal handling (wide character support)
+* `libcap`, *Optional* user-space interfaces to the POSIX 1003.1e
+* `libdl`, if not building static and support for some of the optional libraries is enabled
+* `libsensors`, readout of temperatures and CPU speeds.
+* `systemd`
```

Comment:
This one is optional. TBH, this was actually the first library we explicitly included in a way so we can run without it.